README.md

```diff
@@ -1,36 +1,67 @@
-# GSoft.Authentication.Client.Credentials.Grant
+# GSoft.Authentication.ClientCredentialsGrant
 
 [![nuget](https://img.shields.io/nuget/v/GSoft.Authentication.Client.Credentials.Grant.svg?logo=nuget)](https://www.nuget.org/packages/GSoft.Authentication.Client.Credentials.Grant/)
 [![build](https://img.shields.io/github/actions/workflow/status/gsoft-inc/gsoft-authentication-client-credentials-grant/publish.yml?logo=github&branch=main)](https://github.com/gsoft-inc/gsoft-authentication-client-credentials-grant/actions/workflows/publish.yml)
```

Comment:
I wonder if we should rename the github repo
* from `gsoft-authentication-client-credentials-grant`
* to `gsoft-authentication-clientcredentialsgrant` ?

If we want to do that, use VS Code to ensure we find & replace any reference

Reply:
It's done!